exercises/alphametics/example.rb

```diff
@@ -0,0 +1,41 @@
+class Alphametics
+  VERSION = 1
+
+  def solve(puzzle)
+    letters = Hash.new(0)
+    puzzle = format_to_ruby(puzzle)
```

Comment:
The tests have cases like `R ^ 2`, so this would it to the equivalent Ruby expression `R ** 2`. Would `math_operators_to_ruby` be more meaningful?